python-javabridge/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: python-javabridge
+  version: "4.0.0"
+
+source:
+  fn: python-javabridge-4.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/58/5c/bfcc52002c9dfe5fe3bd067783243ab0130f59f757b0517f62f1ba7e88ea/python-javabridge-4.0.0.tar.gz
+  md5: d2cb87302155901c370bc3fa6e58edd2
+
+requirements:  # there are many more requirements (boto3, botocore, future jmespath, numpy, python-dateutil, python-javabridge, s3transfer, six, urllib3) but we bank on python-bioformats pulling these for us
```

Comment:
requirements should be specified where they are needed so there is a clear dependency chain - this package should stand alone and shouldn't depend on python-bioformats

Reply:
I thought I'd deleted this comment. It was from an earlier thing I was testing.